scripts/create_superuser.sh

```diff
@@ -26,6 +26,11 @@ usage() {
 }
 
 create_user() {
+    if ! command -v poetry &>/dev/null; then
+        log-warn "poetry is not installed, please install poetry(version >= 1.4.0) first."
```

Comment:
We upgrade to poetry 2.x, probably not an issue for `run` sub command but I would require it anyway for safeness and consistency. 
```suggestion
        log-warn "poetry is not installed, please install poetry(version >= 2.0 ) first."
```

Reply:
Thanks for attending my suggestions. And thanks for your contribution.